pom.xml

```
@@ -1204,10 +1204,10 @@
                 </requireMavenVersion>
                 <requireProperty>
                   <regexMessage>
-		     x86_64/AARCH64/PPCLE64/s390x_64 JDK must be used.
+		     x86_64/AARCH64/PPCLE64/s390x_64/loongarch64 JDK must be used.
                   </regexMessage>
                   <property>os.detected.arch</property>
-                  <regex>^(x86_64|aarch_64|ppcle_64|s390_64)$</regex>
+                  <regex>^(x86_64|aarch_64|ppcle_64|s390_64|loongsrch_64)$</regex>
```

```suggestion
                  <regex>^(x86_64|aarch_64|ppcle_64|s390_64|loongarch_64)$</regex>
```
This might be a typo?